problems/arraysWithLoops.js

```diff
@@ -25,39 +40,75 @@ function shoutWhileLoop() {}
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForOfLoop() {}
+function shoutForOfLoop(array) {
+  let newArray = []
+  for (let i of array){
+    newArray.push(i + "!")
+    i++;
```

Comment:
"for of" loops increment themselves so you don't need i++

also remember i isn't an index in a for of loop, it's the element, so this is like saying "happy"++